sklearn/linear_model/_cd_fast.pyx

```diff
@@ -694,17 +735,32 @@ def enet_coordinate_descent_gram(floating[::1] w,
 
 
 def enet_coordinate_descent_multi_task(
-        floating[::1, :] W, floating l1_reg, floating l2_reg,
-        np.ndarray[floating, ndim=2, mode='fortran'] X,  # TODO: use views with Cython 3.0
-        np.ndarray[floating, ndim=2, mode='fortran'] Y,  # hopefully with skl 1.0
-        int max_iter, floating tol, object rng, bint random=0):
+    floating[::1, :] W,
+    floating l1_reg,
+    floating l2_reg,
+    np.ndarray[floating, ndim=2, mode='fortran'] X,  # TODO: use views with Cython 3.0
+    np.ndarray[floating, ndim=2, mode='fortran'] Y,  # hopefully with skl 1.0
```

Comment:
I guess the comment there can be removed (as it is kind of a general improvement which can be done in the codebase) or rephrased. The suggestion here applies in other places.
```suggestion
    # TODO: use const qualified fused-typed memoryviewwhen Cython 3.0 is used.
    np.ndarray[floating, ndim=2, mode='fortran'] X,
    np.ndarray[floating, ndim=2, mode='fortran'] Y,
```